src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -1412,10 +1414,21 @@ treatAsInline = Set.fromList
   , "pagebreak"
   ]
 
+dolabel :: PandocMonad m => LP m Inlines
+dolabel = do
+  v <- braced
+  return $ spanWith ("",[],[("data-label", toksToString v)]) $ inBrackets $ str $ toksToString v
```

Comment:
Yes, because `setCaption` uses `"data-label"` as well. I wanted to keep the same behavior here. Should I change both functions to use just `"label"`